packages/zoe/src/zoeService/startInstance.js

```diff
@@ -61,6 +63,28 @@ export const makeStartInstance = (
 
     const instance = makeInstanceHandle();
 
+    // Invitations whose descriptions contain any of the strings will be blocked
+    /** @type {Readonly<string[]>} */
+    let offerFilterStrings = [];
+    // if any string in the list matches, don't process the invitation
+    const isBlocked = string =>
+      offerFilterStrings.some(s => string.indexOf(s) > -1);
```

Comment:
This is the part I don't get: Why substring rather than exact match? Do you have a use case in mind?